gateway/src/inference/providers/gcp_vertex_gemini.rs

```diff
@@ -98,6 +69,40 @@ pub fn default_api_key_location() -> CredentialLocation {
 pub enum GCPVertexCredentials {
     Static(GCPServiceAccountCredentials),
     Dynamic(String),
+    #[cfg(any(test, feature = "e2e_tests"))]
+    None
+}
+
+impl TryFrom<(Credential, &str)> for GCPVertexCredentials {
+    type Error = Error;
+    
+    fn try_from((credentials, model): (Credential, &str)) -> Result<Self, Error> {
+        match credentials {
+            Credential::Static(value) => Ok(GCPVertexCredentials::Static(
+                GCPServiceAccountCredentials::from_path(value.expose_secret()).map_err(|e| {
+                    Error::new(ErrorDetails::GCPCredentials {
+                        message: format!("Failed to load GCP credentials: {}", e),
+                    })
+                })?,
+            )),
```

Comment:
I think we should probably not have this -- let's just do FileContents and Dynamic for now